src/main/kotlin/bloombox/client/test/ClientRPCTest.kt

```diff
@@ -75,9 +74,17 @@ open class ClientRPCTest {
    * close it afterwards.
    */
   fun withClient(block: (RPCClient) -> Unit) {
+    withClient(null, block)
+  }
+
+  /**
+   * Provide a callback with a setup client, and then immediately
+   * close it afterwards.
+   */
+  fun withClient(settings: Bloombox.Settings? = null, block: (RPCClient) -> Unit) {
     var client: RPCClient? = null
     try {
-      client = RPCClient()
+      client = RPCClient(settings)
```

Comment:
made `settings` in `ClientRPCTest` overridable, moved it to the main library so we can access it in platform tests